cwltool/main.py

```diff
@@ -42,20 +58,34 @@
 from .stdfsaccess import StdFsAccess
 from .update import ALLUPDATES, UPDATES
 from .utils import onWindows, windows_default_container_id
+from ruamel.yaml.comments import Comment, CommentedSeq, CommentedMap
 
 _logger = logging.getLogger("cwltool")
 
-
-def single_job_executor(t,  # type: Process
+#Adding default namespaces
+document = ProvDocument()
+engineUUID=""
+activity_workflowRun={}  # type: Dict[Text, Any]
+defaultStreamHandler = logging.StreamHandler()
```

Comment:
Consider reusing `defaultStreamHandler` from `loghandler` module

Reply:
Apparently I did not need it :). Thanks for indicating. removed in new commits.